maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -287,6 +287,14 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun fetchSession(
+        request: MaestroAndroid.SessionRequest,
+        responseObserver: StreamObserver<MaestroAndroid.SessionResponse>
+    ) {
+        responseObserver.onNext(sessionResponse { id = SdkService.sessionInfo.sessionId })
```

Comment:
Can session id be null?

Reply:
No, the sessionId is the first thing created when the users uses the SDK

Reply:
👍